lib/matplotlib/tests/test_lines.py

```diff
@@ -133,5 +133,12 @@ def test_nan_is_sorted():
     assert_true(not line._is_sorted(np.array([1,np.nan,3])))
 
 
+def test_invalid_marker():
+    # Specifying an invalid marker should raise a good exception.
+    line = mpl.lines.Line2D([], [])
+    assert_raises_regexp(ValueError, "Unrecognized marker style \"7\"",
+                         line.set_marker, u'7')
```

Comment:
Inclined to agree. I was hoping to actually test the unicode definition, but hey-ho, it is harder to test.